index.html

```diff
@@ -1,47 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<link rel="stylesheet" type="text/css" href="css/styles.css">
-	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Tribute to Steve Jobs</title>
-</head>
-<body>
+  <head>
+    <link rel="stylesheet" type="text/css" href="css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute to PELE</title>
+  </head>
+  <body>
     <main id="main">
       <div id="heading" class="center">
         <h1 id="title" class="center">Pele</h1>
         <p class="font"><i>The greatest football player of all time</i></p>
       </div>
       <div id="img-div">
-        <img class="center" id="image" src="https://cdn.britannica.com/38/211838-050-3D3D2391/Pele-Brazilian-soccer-player-athlete-circa-1970s.jpg" alt="Pele">
+        <img
+          class="center"
+          id="image"
```

Comment:
Please try to avoid using ambiguous class names and tag ID's, in your case _center, image, font_ , this will help you to write clean, reusable and maintainable code.